### PR TITLE
Register variable for later use

### PR DESCRIPTION
**Pull request by haad**
Register variable which can be used later in a playbook to test if rules where changed.

### CODE REVIEW

**Comment by jtyr** [comment]
Thanks for your PR, @haad. Please could you explain for what exactly you want to use the variable later in the playbook?

**Comment by haad** [comment]
To be able to detect a change. In which case i have to reboot server to
apply new settings.

File: tasks/main.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@
   template:
     src: rules.j2
     dest: /etc/udev/rules.d/70-persistent-net.rules
+  register: udev_rename_netiface
```
Comment (jtyr) [inline tasks/main.yaml]: Please rename the variable to `udev_rename_netiface_config_changed`.

**Review by jtyr — CHANGES_REQUESTED** [review]

File: tasks/main.yaml (hunk 1 above)
Comment (haad) [reply tasks/main.yaml]: Should be ok now

**Review by jtyr — APPROVED** [review]

**Comment by jtyr** [comment]
Merged. Thanks for your PR.

**Pull request closed without merging**